### PR TITLE
feat: Update contact page and navbar

### LINKED ISSUE #14: 🚀 Feature: Propose contact page
### Description

Please propose a solution for contact page design.

### Use Case

Critical UX for users

### Proposed Solution

N/A

### Alternatives

N/A

### Additional Information

N/A

### PR DESCRIPTION
**Pull request by sadeepabandara**
# Pull Request

**Description**

Create the contact form, update the navbar and background gradient

**Related Issue(s)**

Closes https://github.com/nibmcs/nibmcs.org/issues/14

**Checklist**

Before submitting this pull request, please ensure the following:

- [x] I have read and followed the project's contributing guidelines.
- [x] My code follows the project's coding style and conventions.
- [x] My commits follow the project's commit message guidelines.
- [x] I have reviewed my changes to ensure they are complete and accurate.

**Submission**

Before submitting, please review your pull request to ensure it is complete and follows the provided checklist. Thank you for contributing!

### CODE REVIEW

**Review by dileepadev-old — APPROVED** [review]
👍🏼 works fine!

File: src/app/contact/page.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        <div className='container flex items-center justify-center h-full mx-auto text-center md:py-4 xl:text-left'>
+          <div className='flex flex-col w-full max-w-[700px]'>
+            <motion.h2
+              variants={fadeIn('up', 0.2)}
+              initial='hidden'
+              animate='show'
+              exit='hidden'
+              className='mb-12 text-center text-[32px] md:text-[48px] leading-tight font-semibold'
+            >
+              Let's <span className='text-[#1E50FF]'>connect.</span>
```
Comment (dileepadev-old) [inline src/app/contact/page.tsx]: 76:18  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

File: src/app/contact/page.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                rows={10}
+                placeholder='Message'
+                className='h-[180px] resize-none p-6 rounded-lg w-full bg-transparent border outline-none focus:ring-1 focus:ring-[#1E50FF] border-white/20 placeholder:text-white/30 placeholder:font-light'
+              ></textarea>
+              <button
+                type='submit'
+                className='border rounded-full h-[52px] border-white/50 max-w-[170px] px-8 transition-all duration-300 flex items-center justify-center overflow-hidden hover:border-[#1E50FF] group'
+              >
+                <span className='group-hover:-translate-y-[120%] group-hover:opacity-0 transition-all duration-500'>
+                  Let's talk
```
Comment (dileepadev-old) [inline src/app/contact/page.tsx]: 120:22  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

**Review by dileepadev-old — CHANGES_REQUESTED** [review]
Check line number 76 & 120 @sadeepabandara

**Review by dileepadev-old — APPROVED** [review]
👍🏼

**Comment by sadeepabandara** [comment]
@dileepabandara I have one change todo please hold the PR merge

**Review by dileepadev-old — APPROVED** [review]
🔥 Nice & Cool 👍🏼

**Pull request merged**